pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -940,84 +939,76 @@ func TestTopologyAwareAllocateCPUs(t *testing.T) {
 // above test cases are without kubelet --reserved-cpus cmd option
 // the following tests are with --reserved-cpus configured
 type staticPolicyTestWithResvList struct {
-	description         string
-	topo                *topology.CPUTopology
-	numReservedCPUs     int
-	reserved            cpuset.CPUSet
-	featureGatesEnabled []featuregate.Feature
-	cpuPolicyOptions    map[string]string
-	stAssignments       state.ContainerCPUAssignments
-	stDefaultCPUSet     cpuset.CPUSet
-	pod                 *v1.Pod
-	expErr              error
-	expNewErr           error
-	expCPUAlloc         bool
-	expCSet             cpuset.CPUSet
+	description      string
```

Comment:
thanks for the simplification. Please squash this commit in the one which is adding the unit tests.